src/components/sbb-pearl-chain-vertical/sbb-pearl-chain-vertical.stories.js

```diff
@@ -0,0 +1,687 @@
+          10:31
+        </div>
+      </sbb-pearl-chain-item>
+      <sbb-pearl-chain-item pearlChainItemAttributes={args}>
+        <div
+          slot="right"
+          style={{
+            marginTop: '-10px',
+            marginInlineStart: '10px',
+          }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              gap: '100px',
+            }}
+          >
+            <div> Haltestelle</div>
+            <div>Gleis 12</div>
+          </div>
+          <div style={{ paddingBottom: '5px', paddingTop: '5px' }}>
+            <span>
+              1.<sbb-icon name="utilization-high"></sbb-icon> 2.
+              <sbb-icon name="utilization-high"></sbb-icon>
+            </span>
+          </div>
+        </div>
+        <div slot="left" style={{ marginTop: '-10px', marginInlineEnd: '10px' }}>
+          <div style={{ fontWeight: 'bold' }}>19:00</div>
+          <div style={{ marginTop: '40px' }}>10:31</div>
+        </div>
+      </sbb-pearl-chain-item>
+      <sbb-pearl-chain-item
+        pearlChainItemAttributes={{
+          lineType: 'standard',
+          lineColor: 'charcoal',
+          dotColor: 'charcoal',
+          minHeight: '89',
+          hideLine: false,
+          dotType: 'thick-bullet',
+          dotSize: 'small',
+        }}
+      >
+        <div
+          slot="right"
+          style={{
+            marginTop: '-10px',
+            marginInlineStart: '10px',
+          }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              gap: '100px',
+            }}
+          >
+            <div> Haltestelle</div>
+            <div>Gleis 12</div>
+          </div>
+          <div style={{ paddingBottom: '5px', paddingTop: '5px' }}>
+            <span>
+              1.<sbb-icon name="utilization-high"></sbb-icon> 2.
+              <sbb-icon name="utilization-high"></sbb-icon>
+            </span>
+          </div>
+        </div>
+
+        <div slot="left" style={{ marginTop: '-10px', marginInlineEnd: '10px' }}>
+          <div style={{ fontWeight: 'bold' }}>19:00</div>
+          <div style={{ marginTop: '40px' }}>10:31</div>
+        </div>
+      </sbb-pearl-chain-item>
+      <sbb-pearl-chain-item
+        pearlChainItemAttributes={{
+          lineType: 'thin',
+          lineColor: 'metal',
+          dotColor: 'charcoal',
+          minHeight: '89',
+          hideLine: false,
+          dotType: 'standard',
+          dotSize: 'medium',
+        }}
+      >
+        <div
+          slot="right"
+          style={{
+            marginTop: '-10px',
+            marginInlineStart: '10px',
+          }}
+        >
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              gap: '100px',
+            }}
+          >
+            <div> Haltestelle</div>
```

Comment:
Examples should generally be in german language (There are many other german occurrences)

Reply:
Fixed comment for english, as was intended by @jeripeierSBB